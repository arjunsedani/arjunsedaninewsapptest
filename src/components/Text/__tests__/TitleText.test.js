/*
 * Created Date: February 15th 2019, 3:06:44 pm
 * Copyright (c) 2019 Arjun Sedani
 *
 * @flow
 */

import React from 'react';
import renderer from 'react-test-renderer';

import { TitleText, BodyText } from '..';

const setup = () => {
  const tree = renderer.create(<TitleText />);

  return { tree };
};

const setupBodyText = () => {
  const tree = renderer.create(<BodyText />);

  return { tree };
};

describe('Title Text', () => {
  const { tree } = setup();
  it('renders component to snapshot', () => {
    expect(tree).toMatchSnapshot();
  });
});

describe('Body Text', () => {
  const { tree } = setupBodyText();
  it('renders component to snapshot', () => {
    expect(tree).toMatchSnapshot();
  });
});
