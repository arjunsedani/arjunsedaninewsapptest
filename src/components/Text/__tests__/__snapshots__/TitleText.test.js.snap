// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body Text renders component to snapshot 1`] = `
<Text
  style={
    Array [
      Object {
        "alignSelf": "stretch",
        "color": "#002124",
        "fontSize": 14,
        "textAlign": "left",
      },
    ]
  }
/>
`;

exports[`Title Text renders component to snapshot 1`] = `
<Text
  style={
    Array [
      Object {
        "fontSize": 20,
        "textAlign": "center",
      },
    ]
  }
/>
`;
